config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -12,7 +12,7 @@ periodics:
   spec:
     serviceAccountName: prowjob-default-sa
     containers:
```

Comment:
We need the same change in: cluster-api-periodics-main-upgrades.yaml

P.S. the prior art PR link in the issue was wrong. Should have been: https://github.com/kubernetes/test-infra/pull/27326/files